core/sequencer/sequencer.go

```diff
@@ -373,3 +445,21 @@ func queueLogLeaf(ctx context.Context, tlog trillian.TrillianLogClient, logID in
 	}
 	return nil
 }
+
+// RegisterMutationsChannel registers a mutations channel created by the
+// sequencer service handler.
+func (s *Sequencer) RegisterMutationsChannel(ch chan *tpb.GetMutationsResponse) {
+	s.mMux.Lock()
+	defer s.mMux.Unlock()
+	s.mChannels = append(s.mChannels, ch)
+}
+
+// disseminateMutations sends mutations to sequencer service handler through the
+// registered channels.
+func (s *Sequencer) disseminateMutations(m *tpb.GetMutationsResponse) {
+	s.mMux.Lock()
+	defer s.mMux.Unlock()
+	for _, ch := range s.mChannels {
+		ch <- m
```

Comment:
The questions I'm asking here: 
1) Do we have a way of knowing that `ch` won't block, or be closed? (sending to a closed channel causes panic)
2) Do we have a way of allowing clients to unsubscribe when they disconnect?